### PR TITLE
:bug: Fix lint issues: gofumpt linter

### PR DESCRIPTION
**Pull request by chrismcgehee**
* **Please check if the PR fulfills these requirements**
- [ n/a] Tests for the changes have been added (for bug fixes / features)
- [x] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Code quality improvement

* **What is the current behavior?** (You can also link to an open issue here)
This PR fixes issues from the [gofumpt](https://github.com/mvdan/gofumpt) linter (various style rules).

* **What is the new behavior (if this is a feature change)?**
n/a

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: checks/branch_protected_test.go [hunk 1]
```diff
@@ -46,40 +46,42 @@ func TestIsBranchProtected(t *testing.T) {
...
 		{
 			name: "Nothing is enabled",
-			args: args{protection: &github.Protection{
-				RequiredStatusChecks: &github.RequiredStatusChecks{
-					Strict:   false,
-					Contexts: nil,
-				},
-				RequiredPullRequestReviews: &github.PullRequestReviewsEnforcement{
-					DismissalRestrictions: &github.DismissalRestrictions{
+			args: args{
```
Comment (chrismcgehee) [inline checks/branch_protected_test.go:49]: Changes in this file are purely whitespace formatting.

**Comment by chrismcgehee** [comment]
The Validate job failed here because the differing test cases made the `dupl` linter fail due to repeated code. In my opinion for test cases like this, it's perfectly fine to repeat code. If you agree with this, I'll add `nolint` directives to this PR.

**Review by naveensrinivasan — APPROVED** [review]
Thanks!

**Comment by naveensrinivasan** [comment]
> The Validate job failed here because the differing test cases made the `dupl` linter fail due to repeated code. In my opinion for test cases like this, it's perfectly fine to repeat code. If you agree with this, I'll add `nolint` directives to this PR.

Sounds good. Please do it. Thanks

**Comment by chrismcgehee** [comment]
I believe the Validate job failed this time due to the issue in PR [#392](https://github.com/ossf/scorecard/pull/392). Once that one is merged, this PR should be good to merge.

**Pull request merged**